src/components/dashboard/sidebar/hooks/useListitems.js

```diff
@@ -30,26 +32,26 @@ const useListItems = (drawerOpen, drawerSetter) => {
       setter(true)
     }
 
-  const renderCollapsableList = (listItem, baseUrl, subList, opener, setter) => {
+  const renderCollapsableList = (listItem, baseUrl, subList, isOpen, setter) => {
     if (!subList) {
       return [];
     }
 
     return (
       <Fragment key={listItem.id}>
         <Tooltip key={listItem.id} title={listItem.title} disableInteractive>
-          <ListItemButton key={listItem.id} onClick={() => {!drawerOpen ? toggleDrawerAndList(setter) : setter(!opener)}}>
+          <ListItemButton key={listItem.id} onClick={() => {!drawerOpen ? toggleDrawerAndList(setter) : setter(!isOpen)}}>
               <ListItemIcon>
-                {opener ? <ExpandLess /> : <ExpandMore />}
+                {isOpen ? <ExpandLess /> : <ExpandMore />}
```

Comment:
Nice change!

Reply:
I read `opener` and wanted to berate younger me lol

Reply:
It is always a good sign if you read code you wrote before and you find things you want to change!